### PR TITLE
Improve README readability

### PR DESCRIPTION
**Pull request by Mogztter**
## :bookmark_tabs: Summary

Improve the readability of the README:

- Fix link to the changelog
- Remove changelog from the readme
- Use titles sparingly
- Add emojis! :tada:

## :straight_ruler: Design Decisions

In my opinion, the README is hard to parse because there are too many bold paragraphs and titles.

### :clipboard: Tasks

Make sure you
- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md) 
- [x] :computer: have added unit/e2e tests (not applicable)
- [x] :bookmark: targeted `develop` branch

### CODE REVIEW

**Comment by Mogztter** [comment]
Here's the result:

| Current | Proposal |
| ------- | -------- |
| about-current | about |

**Comment by NeilCuzon** [comment]
Looks great! I love the emojis.

**Comment by Mogztter** [comment]
Thanks @NeilCuzon!

I would like to improve the documentation, published at https://mermaid-js.github.io/mermaid, but I'm not sure where the content is stored.
I found several "copies" of the documentation:

- https://github.com/mermaidjs/mermaidjs.github.io
- https://github.com/mermaid-js/mermaid/tree/develop/docs
- https://github.com/mermaid-js/mermaid-js.github.io
- https://github.com/mermaid-js/docs
- https://github.com/mermaidjs/mermaidjs.github.io
- https://github.com/mermaidjs/mermaid-gitbook

I would recommend to archive deprecated repositories to make it easier to contribute documentation.

**Comment by NeilCuzon** [comment]
the content on display here: https://mermaid-js.github.io/mermaid, are auto-generated from the docs in the master branch.
it does have some sister projects and those could be the other repositories in the project.  

there is a shortage of hands at the moment, your help would be very much welcome. 

your earlier contribution was in the right place actually, @knsv is generally the one who merges the develop and master branches when there is a new version for release. 

if you would like, you can make some edits to the docs here: https://github.com/mermaid-js/mermaid/tree/develop/docs

**Comment by Mogztter** [comment]
Thanks for the clarification.
I will do my best to help 😉
